src/modules/addScore.js

```diff
@@ -0,0 +1,28 @@
+const myForm = document.getElementById('add-score-form');
+
+const addScore = () => {
+  const key = 'yAtQ05XYzJj2yusufJCO';
+  const base = `https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${key}/scores`;
+
+  myForm.addEventListener('submit', async (e) => {
+    e.preventDefault();
+    const name = document.getElementById('name').value;
+    const score = document.getElementById('score').value;
+
+    await fetch(base, {
+      method: 'POST',
+      body: JSON.stringify({
+        user: name,
+        score,
+      }),
+      headers: {
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+    })
+      .then((response) => response.json());
```

Comment:
- [ ] [OPTIONAL] There will be no need using the `.then` if you use the `await` keyword. You can do something like this instead.

```js
const res = await fetch(base, {
      method: 'POST',
      body: JSON.stringify({
        user: name,
        score,
      }),
      headers: {
        'Content-type': 'application/json; charset=UTF-8',
      },
    })

const data = await res.json()
```

Reply:
okay thank you so much @AnselemOdims 👍🏼